easylist/easylist_adservers.txt

```diff
@@ -7184,6 +7185,7 @@
 ||kliksaya.com^
 ||kliktrkr.com^
 ||klipmart.com^
+||kliqz.com^
```

Comment:
That was an old one... ~~https://archive.mypdns.org/T4672~~

![image](https://user-images.githubusercontent.com/44526987/140593640-325274c7-d9e1-43c1-9b8c-36a9aefdd2f9.png)